modules/end-user-guide/pages/url-parameter-for-the-ide.adoc

```diff
@@ -0,0 +1,52 @@
+:_content-type: CONCEPT
+:description: URL parameter for the IDE
+:keywords: ide, editor, workspace-ide, workspace-editor
+:navtitle: URL parameter for the IDE
+:page-aliases: url-parameter-for-the-in-browser-ide.adoc
+
+[id="url-parameter-for-the-ide"]
+= URL parameter for the IDE
+
+If the URL for starting a new workspace doesn't contain a parameter specifying the integrated development environment (IDE), the workspace loads with the default IDE, which is Microsoft Visual Studio Code - Open Source.
+
+The URL parameter for specifying a supported IDE is `che-editor=`. You can use it to start a workspace with a clone of the same project repository and your choice of a supported IDE.
```

Comment:
I'd shift the focus of the first sentence to what the user can do.

Reply:
Thank you. The reason it's phrased as a definition sentence is because this is a concept/reference module. I tried to avoid phrasing it as a procedure.